jenkins-pipelines/longevity_mv_synchronous_updates.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
```

Comment:
I think we should rename the pipeline to `features-*` so it would go under features, and no under longevity

Reply:
@roydahan do we want to change all MV longevities to `features`?